otlp/traces.go

```diff
@@ -146,6 +150,7 @@ func TranslateTraceRequest(request *collectorTrace.ExportTraceServiceRequest, ri
 					events = append(events, Event{
 						Attributes: attrs,
 						Timestamp:  timestamp, // use timestamp from parent span
+						SampleRate: sampleRate,
```

Comment:
@robbkidd and I talked about this some yesterday.  Is it safe to assume that the events and links on the span being sample are representative?  Is this a meaningful/safe assumption?

Reply:
Since that conversation, I'm currently thinking "yes".

In a world where traces have been told to sample only 1/N of themselves, everything that occurs within a sampled trace is representative of N others.

Accurate? No. But that's head-sampling for you.

Reply:
We have talked about this, and we agree that this strategy is the best way to handle this right now. Sampling is a dark art, and you can always construct scenarios under which it will fail, so we feel that consistency here is the best answer in the general case.